scripts/tests/change-readme.test.js

```diff
@@ -0,0 +1,36 @@
+const { expect } = require('chai')
```

Comment:
Snapshot tests would be better for this.

Reply:
I've done some reading into snapshot testing, but I'm not sure how to apply this here, can you give me some more context on how to do this here please?

Reply:
Sure, so for a function that manipulates a string or generates a string in multiple ways, it can sometimes be cumbersome to write "expected" tests for it, they lose readability and covering multiple cases is complicated. 

Snapshot tests can be added to such a function as follows: 

Instead of writing

```ts
expect(
  getNewReadMeContent({
    releaseChannel: 'latest',
    npmVersion: '2.4.0',
    githubActionContextSha: 'sha'
  })
).to.be('expected string')
```

You can do

```ts
expect(
  getNewReadMeContent({
    releaseChannel: 'latest',
    npmVersion: '2.4.0',
    githubActionContextSha: 'sha'
  })
).toMatchSnapshot() // Jest syntax
```

This test case will always pass the first time and produce a snapshot file i.e. output of this test. You need to manually verify that file to see if that is correct. If yes, you can commit the test and snapshot. 

In future, if there are any unexpected changes to that file, the test would fail and return the diff of new result with the snapshot file. 

If it is bug, you can fix it but if the change is indeed expected (new feature in readme change say), you need to run `yarn test -u` (jest syntax) to update the snapshots. 

Since we already have a test for this, you can create an issue for that and do it in a separate PR. It is also a matter of preferences. Some people like this way of testing more :)

Reply:
Ah got it, thanks for the explanation 🙏 🚀 I've created an issue for this.